src/lib/registration/registration_codes_loader.rb

```diff
@@ -54,6 +54,18 @@ def reg_codes_from_usb_stick
       nil
     end
 
+    # Loades registration code from /etc/install.inf
+    #
+    # Expected format is reg_code: <product>:<registration_code>
+    # for example: reg_code: sle15:abcdefghijkl
+    #
+    # @return [Hash{String => String, nil] The key of the hash is product name
```

Comment:
The type annotation is missing a `}`, and it does matter where we put it.
For consistency with the other methods the return type should be a `Hash{String => String}, nil`, not a `Hash{String => String, nil}` so we should return `... : nil` and the caller should use `reg_codes_from_foo || reg_codes_from_bar || {}`.

Reply:
ok